File: Backend/Model/Entities/Notification.cs
namespace Backend.Model.Entities
{
    public class Notification : BaseEntity
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Message { get; set; }
        public string? Route { get; set; }
        public string? Type { get; set; }
        public string? SenderUserId { get; set; }
        public string? SenderDisplayName { get; set; }
        public int? TournamentId { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: Frontend/FrontendApp/src/app/model/notification.ts
export interface NotificationDto {
  notificationId: number;
  title: string;
  message: string;
  route?: string;
  type?: string;
  senderUserId?: string;
  senderDisplayName?: string;
  tournamentId?: number;
  createdAt: string;
  isRead: boolean;
}

  
